nodejs/webrtc-hello-world/src/server.ts

```diff
@@ -282,11 +281,8 @@ const callPhone = async (phoneNumber: string, participant: Participant) => {
  * @param deviceToken device token received from the call control API for a participant
  */
 const generateTransferBxml = async (deviceToken: string) => {
-  //Get the tid out of the participant jwt
-  var decoded: any = jwt_decode(deviceToken);
-
-  return `<?xml version="1.0" encoding="UTF-8" ?>
-    <Response>
-      <Transfer transferCallerId="${decoded.tid}"><PhoneNumber>${sipxNumber}</PhoneNumber></Transfer>
-    </Response>`;
+  return '<?xml version="1.0" encoding="UTF-8" ?>\n'
```

Comment:
I tried including the WebRTC SDK in order to use the JS helper but it doesn't include a declarations file and thus won't work with Typescript. I'll work with DevX on getting that fixed but in the meantime I elected to include the XML directly.